src/components/results/ResultBody.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const ResultBody = ({ filteredResult }) => {
+    return (
+        <tbody>
+            {filteredResult.map((data, i) =>
+                <tr key={i}>
+                    <td>{data.title}</td>
+                    <td>{data.firstName}</td>
+                    <td>{data.surname}</td>
+                    <td>{data.email}</td>
+                    <td>{data.roomId}</td>
+                    <td>{data.checkInDate}</td>
+                    <td>{data.checkOutDate}</td>
+                    <td>{parseInt((new Date(data.checkOutDate) - new Date(data.checkInDate)) / (86400000), 10)} days</td>
```

Comment:
Oh I forgot to say - I guess this works, I wouldn't say it's very readable. I would recommend looking at installing moment.js and using the `.diff` method